app/models/variant.rb

```diff
@@ -42,6 +43,10 @@ def init
     self.civic_actionability_score ||= 0 if self.has_attribute? :civic_actionability_score
   end
 
+  def update_allele_registry_id
+    SetAlleleRegistryIdSingleVariant.new().perform(self)
```

Comment:
I believe you'll want this to be `SetAlleleRegistryIdSingleVariant.perform_later(self)` otherwise it will execute synchronously rather getting queued in the background.

Reply:
Fixed.